.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```

Comment:
Please do not add any of your IDE files into the project. The entire `.idea` directory shouldn't be present here, there should be no changes except the relevant ones to `yo/main.py`.